rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -260,12 +235,12 @@ namespace
 			mip = D3D12_FILTER_TYPE_LINEAR;
 			return;
 		case CELL_GCM_TEXTURE_CONVOLUTION_MIN:
-			LOG_WARNING(RSX, "CELL_GCM_TEXTURE_CONVOLUTION_MIN not supported, fallback to bilinear filtering");
 			min = D3D12_FILTER_TYPE_LINEAR;
 			mip = D3D12_FILTER_TYPE_POINT;
 			return;
 		}
-		unreachable("Wrong min filter");
+		unreachable("Unimplemented min filter used %x, please report this to a developer.", min_filter);
+
```

Comment:
@Zangetsu38 Can you even read? No, ofcourse not! Who am I kidding - you use Google Translate...
He was suggesting to shorten the message to something more simple, like:
"Unknown min filter: 0x%x, please report this to a developer."

Honestly my suggestion would be: go take ~~some~~ many English courses.